tasks/check_space.yml

```
@@ -1,10 +1,32 @@
 ---
-- name: Verify that there is enough storage space
-  ansible.builtin.script: >-
-    snapshot.py check -a -r {{ snapshot_lvm_percent_space_required | quote }}
-    -p {{ snapshot_lvm_prefix | quote }} -s {{ snapshot_lvm_suffix | quote }}
-  register: snapshot_cmd
-  args:
-    executable: "{{ __snapshot_python }}"
-  failed_when:
-    snapshot_cmd.rc != 0
+- name: Construct cmd and execute
+  block:
+    - name: Show __cmd
+      vars:
+        __cmd: "{{ 'snapshot.py check ' ~
+          ('-a ' if snapshot_lvm_all_vgs == true else '') ~
```

```suggestion
          ('-a ' if snapshot_lvm_all_vgs else '') ~
```
Ansible prefers not to use `boolean_var == true` or `boolean_var == false`